cms/static/sass/elements/_footer.scss

```diff
@@ -58,5 +62,42 @@
         }
       }
     }
+
+    .footer-content-secondary {
+      @include clearfix();
+      margin-top: $baseline;
+    }
+
+    .footer-about-copyright, .footer-about-openedx {
+      display: inline-block;
+      vertical-align: middle;
+    }
+
+    // platform trademarks
+    .footer-about-copyright {
+      width: flex-grid(4, 12);
+      @include float(left);
+      margin-right: flex-gutter(2);
```

Comment:
It actually seems to be working fine in Studio wrt RTL as-is. The only issues I'm seeing are with the LMS.

Studio footer for reference:

![screen shot 2015-03-25 at 1 44 08 pm](https://cloud.githubusercontent.com/assets/1985317/6831185/0c54bd9c-d2f5-11e4-8eed-5ff8959a1ac5.png)